DiceKit/FrequencyDistribution.swift

```diff
@@ -190,13 +207,13 @@ extension FrequencyDistribution {
     
     /// This is a special case of `power(x: FrequencyDistribution)`,
     /// for when `x` is `FrequencyDistribution([x: 1])`.
-    public func power(x: Int) -> FrequencyDistribution {
+    public func power(x: Outcome) -> FrequencyDistribution {
         guard x != 0 else { return .multiplicativeIdentity }
         
         // Crappy implementation. Currently O(n). Can be O(log(n)).
         // TODO: Handle negative properly
         var freqDist = self
-        for _ in 1..<abs(x) {
+        for _ in Outcome.multiplicativeIdentity..<abs(x) {
```

Comment:
I'm gonna leave this for now. It gets where when I try to make a frequency distribution of `Successfulness` values (PR forthcoming). I think when I fix the `TODO` of handling negative values, it will all be resolved.